pysat/tests/test_constellation.py

```diff
@@ -577,3 +575,91 @@ def test_to_inst_mult_pad_clean(self):
                                    list(self.inst[1].index))
 
         return
+
+    @pytest.mark.parametrize("method", ["del", "drop"])
+    def test_delitem_all_inst(self, method):
+        """Test Constellation deletion of data variables from all instruments.
+
+        Parameters
+        ----------
+        method : str
+            String specifying the deletion method
+
+        """
+        # Load the Constellation data
+        self.const.load(date=self.ref_time)
+
+        # Delete the UTS data from all instruments
+        dvar = "uts"
+        if method == "del":
+            del self.const[dvar]
+        else:
+            self.const.drop(dvar)
+
+        # Test that this variable is gone from all Instruments
+        for inst in self.const.instruments:
+            assert dvar not in inst.variables
+
+        # Test that the constellation variable list has been updated
+        for var in self.const.variables:
+            assert var.find(dvar) != 0
+
+        return
+
+    @pytest.mark.parametrize("method", ["del", "drop"])
+    def test_delitem_one_inst(self, method):
+        """Test Constellation deletion of data variables from one instrument.
+
+        Parameters
+        ----------
+        method : str
+            String specifying the deletion method
+
+        """
+        # Load the Constellation data
+        self.const.load(date=self.ref_time)
+
+        # Delete the UTS data from all instruments
```

Comment:
I'm still a little confused on this test. I think it says that it is testing deletion from one instrument in a constellation.
But this part appears to delete a data variable from all of the constellation instruments.

Reply:
I think the comment is wrong, but I also really don't know how the constellation object works. 

Based on the code, it looks like we are telling the constellation to drop the uts from pysat testing (at least, that's what we are checking for). @aburrell, any insight?

Reply:
Test snippet:
```
import pysat
from pysat import constellations
inst = list(constellations.testing.instruments)
const = pysat.Constellation(instruments=inst)
const.load(2009, 1)
const.variables
```
The variables stored in the constellation object are appended with the instrument name (except when they are dims, apparently). This test deletes the pysat_testing uts variable from the const-level list and then checks that it has been disappeared at the inst-level as well.

The dims thing should probably have an issue. Leaning toward that route since we have a number of constellation-related improvements for future milestones.